react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component, Fragment } from 'react';
+import Spinner from 'react-spinkit';
+
+import api from '~services/MatchesService';
+
+import styles from './styles.module.scss';
+
+class Matches extends Component {
+  state = {
+    matches: [],
+    loading: true
+  };
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.getMatches();
+    }, 3000);
+  }
+
+  // recomendacioón de alejo
+  parseResponse = response => {
```

Comment:
This is a way to standard all your api request, but its better if you let it outside this component, in a utils file

Reply:
👆

Reply:
it's okay, the changes of this method have been made in the next branches. 😄🤟🏽